packages/wasm-dpp/src/data_trigger/execute_data_triggers_factory.rs

```diff
@@ -0,0 +1,61 @@
+use dpp::document::validation::state::execute_data_triggers::execute_data_triggers_with_custom_list;
+use itertools::Itertools;
+use js_sys::Array;
+use wasm_bindgen::prelude::*;
+
+use crate::{
+    document_batch_transition::document_transition::DocumentTransitionWasm,
+    state_repository::ExternalStateRepositoryLikeWrapper,
+    utils::{Inner, IntoWasm, WithJsError},
+    DataTriggerWasm,
+};
+
+use super::{
+    data_trigger_execution_context::{
+        data_trigger_context_from_wasm, DataTriggerExecutionContextWasm,
+    },
+    data_trigger_execution_result::DataTriggerExecutionResultWasm,
+};
+
+#[wasm_bindgen(js_name=executeDataTriggers)]
+pub async fn execute_data_triggers_wasm(
+    js_document_transitions: Array,
+    js_context: DataTriggerExecutionContextWasm,
+    js_data_triggers: Array,
+) -> Result<Array, JsValue> {
+    console_error_panic_hook::set_once();
```

Comment:
I don't think this hook is supposed to be here :)

Reply:
fixed, thanks!